src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -125,6 +125,14 @@ std::vector<image_annotation> parse_annotations(
         for (const auto& coord_kv : coordinates) {
 
           const flex_string& coord_key = coord_kv.first.get<flex_string>();
+
+          // Check the type of coordinate
```

Comment:
good job. One more thing, please update the docstring if the type constraint of coordinates is not mentioned.

Reply:
also, don't forget to check the width and height. ;-)

Reply:
It does include width and height ;)

Reply:
Ah, it's a map. Good!